backend/scripts/bootstrap.js

```diff
@@ -23,15 +24,23 @@ client.connect()
 const cryptos = ['BTC', 'ETH', 'XRP', 'LTC', 'BCH', 'USDT', 'EOS', 'BNB', 'BSV', 'TRX']
 const symbol = 'EUR'
 const api_key = argv.api_key;
+const numberOfDays = 60;
+const numberOfHours = 48;
+const numberOfMinuts = 120;
 
 // GET ALL CRYPTOS INFOS
 const getCryptoList = async () => {
     const url = `https://min-api.cryptocompare.com/data/all/coinlist`;
     let query = 'INSERT INTO crypto_list (symbol, fullname, picture_url) VALUES\n';
+  
+    const cryptoListBar = new cliProgress.SingleBar({}, cliProgress.Presets.shades_classic);
+    cryptoListBar.start(cryptos.length, 0);
+  
     await axios.get(url)
         .then(response => {
```

Comment:
wtf ? You use `await` and `.then`. That does not make any sense. use async/await, or promises chain, but not both